addons/hr_timesheet/views/project_task_sharing_views.xml

```diff
@@ -15,9 +15,9 @@
             <xpath expr="//field[@name='child_ids']/tree/field[@name='portal_user_names']" position="after">
                 <field name="allocated_hours" widget="timesheet_uom_no_toggle" sum="Total Allocated Time" optional="hide" column_invisible="not parent.allow_timesheets" invisible="not allow_timesheets"/>
                 <field name="effective_hours" widget="timesheet_uom" sum="Effective Hours" optional="hide" column_invisible="not parent.allow_timesheets" invisible="not allow_timesheets"/>
-                <field name="subtask_effective_hours" string="Sub-tasks Hours Spent" widget="timesheet_uom" sum="Sub-tasks Hours Spent" optional="hide" column_invisible="not parent.allow_timesheets" invisible="not allow_timesheets"/>
-                <field name="total_hours_spent" string="Total Hours" widget="timesheet_uom" sum="Total Hours" optional="hide" column_invisible="not parent.allow_timesheets" invisible="not allow_timesheets"/>
-                <field name="remaining_hours" widget="timesheet_uom" sum="Remaining Hours" optional="hide" decoration-danger="progress &gt;= 1" decoration-warning="progress &gt;= 0.8 and progress &lt; 1" column_invisible="not parent.allow_timesheets" invisible="not allow_timesheets"/>
+                <field name="subtask_effective_hours" string="Sub-tasks Time Spent" widget="timesheet_uom" sum="Sub-tasks Hours Spent" optional="hide" column_invisible="not parent.allow_timesheets" invisible="not allow_timesheets"/>
+                <field name="total_hours_spent" string="Total Time Spent" widget="timesheet_uom" sum="Total Time Spent" optional="hide" column_invisible="not parent.allow_timesheets" invisible="not allow_timesheets"/>
```

Comment:
```suggestion
                <field name="total_hours_spent" widget="timesheet_uom" sum="Total Time Spent" optional="hide" column_invisible="not parent.allow_timesheets" invisible="not allow_timesheets"/>
```
I think we don't need the `string` here, as the label is the same one as on the python field? Same goes for multiple other fields in this PR